### PR TITLE
reference: add docs for unified read pool configs

### PR DESCRIPTION
**Pull request by sticnarf**
# What is changed, added or deleted? 

Add documentations for the configurations of TiKV's unified read pool.

### What is the related PR or file link(s)? 

- [x] Reference link(s):
- https://github.com/tikv/tikv/pull/6585 adds options to enable this feature.
- https://github.com/tikv/tikv/pull/6593 enables the feature by default.
- https://github.com/tikv/tikv/pull/6597 adds some more options to this feature.
- [x] This PR is to align with:
- https://github.com/pingcap/docs/pull/1813
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

### CODE REVIEW

**Comment by sticnarf** [comment]
PTAL @AndreMouche

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 1]
```diff
@@ -89,6 +89,43 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
+
+## readpool.unified
+
+统一处理读请求的线程池相关的配置项。
+以下配置只在 `unify-read-pool` 是 `true` 时生效。
+
+### `min-thread-count`
+
++ 统一处理读请求的线程池最少的线程数量。
++ 默认值：1
```
Comment (AndreMouche) [inline dev/reference/configuration/tikv-server/configuration-file.md]: Do we have a max value?

Comment (sticnarf) [reply dev/reference/configuration/tikv-server/configuration-file.md]: No. It's valid as long as it is less than `max-thread-count` (which is trivial)

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 2]
```diff
@@ -89,6 +89,43 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
+
+### `max-tasks-per-worker`
+
++ 统一处理读请求的线程池中单个线程允许积压的最大任务数量，超出后会返回 Server Is Busy。
++ 默认值：2000
++ 最小值：2
+
 ## readpool.storage
 
 存储线程池相关的配置项。
```
Comment (yikeke) [inline dev/reference/configuration/tikv-server/configuration-file.md]: readpool.storage 和 readpool.coprocessor 标题下，英文版里的这句话“They only take effect when `unify-read-pool` is `false`” 是不是忘记加上来了？

Comment (sticnarf) [reply dev/reference/configuration/tikv-server/configuration-file.md]: 是的，对不起。我加一下。

Comment (sticnarf) [reply dev/reference/configuration/tikv-server/configuration-file.md]: Addressed. PTAL again, thanks!

File: dev/reference/configuration/tikv-server/configuration-file.md (hunk 1 above)
Comment (yikeke) [reply dev/reference/configuration/tikv-server/configuration-file.md]: PTAL @AndreMouche

**Pull request merged**